modules/osdk-cli-reference-cleanup.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-cleanup_{context}"]
+= cleanup
+
+The `operator-sdk cleanup` command destroys and removes resources that were created for an Operator that was deployed with the `run` command.
+
+== packagemanifests
+
+`cleanup packagemanifests` subcommand destroys an Operator that was deployed with OLM by using the `run packagemanifests` command.
+
+.`packagemanifests` arguments
+[options="header",cols="1,3"]
+|===
+|Arguments |Description
+
+|`--include` (string)
+|The file path to Kubernetes resource manifests, such as Role objects and Subscription objects. These supplement or override defaults generated by `run` or `cleanup`.
+
+|`--install-mode` (string)
+|The `OperatorGroup` is created with the specified `InstallMode`. Format: `InstallModeType[=ns1,ns2[, ...]]``
+
+|`--kubeconfig` (string)
+|The file path to a Kubernetes configuration file. Defaults to the location specified by `$KUBECONFIG`, or to default file rules if not set
```

Comment:
Please revise to include ending punctuation and to clarify that KUBECONFIG is an environment variable. Maybe:

"Default: The location specified by the `KUBECONFIG` environment variable or default file rules if the environment variable is not set."

I believe during an earlier review, you indicated that the most frequent format was "Default:" rather than the 4.9 sentence format of "The default value is..."

Also, the description is identical to `modules/osdk-cli-reference-run.adoc`, so please make sure the descriptions are identical.